airflow-core/src/airflow/ui/src/i18n/locales/de/common.json

```diff
@@ -163,22 +177,40 @@
     "to": "Zu"
   },
   "task": {
+    "documentation": "Task Dokumentation",
     "lastInstance": "Letzte Task Instanz",
     "operator": "Operator",
     "triggerRule": "Auslöse-Regel"
   },
   "task_one": "Task",
   "task_other": "Tasks",
+  "taskId": "Task ID",
   "taskInstance": {
     "dagVersion": "Dag Version",
+    "executor": "Ausführungsumgebung",
+    "executorConfig": "Konfiguration der Ausführungsumgebung",
+    "hostname": "Hostname",
+    "maxTries": "Maximale Versuche",
     "pid": "PID",
     "pool": "Pool",
-    "queuedAt": "Wartend seit"
+    "poolSlots": "Pool Belegung",
+    "priorityWeight": "Priorität",
+    "queue": "Warteschlange",
+    "queuedWhen": "Wartend seit",
+    "scheduledWhen": "Geplant ab",
+    "triggerer": {
+      "assigned": "Zugewiesener Abrufumgebung",
```

Comment:
```suggestion
      "assigned": "Zugewiesene Abrufumgebung",
```

Ich hatte gar nie realisiert, dass diese Info in the Task Instanzdetails drin ist, ist das neu in 3.0? 😄 Ich nehme an Abrufsumgebung ist der relevante async Python Prozess im Triggerer? 

![image](https://github.com/user-attachments/assets/5efe7938-1065-4210-803f-a70d00218169)

Reply:
Ja, hier geht es um den Triggerer.